core/src/main/java/io/github/fourlastor/game/demo/round/Round.kt

```diff
@@ -69,6 +69,7 @@ class Round @Inject constructor(private val router: StateRouter, private val sta
         currentFaction.units.asSequence()
             .filter { unitInTurn: UnitInRound -> !unitInTurn.hasActed }
             .forEach { unitInTurn: UnitInRound ->
+                Consumer { unitInRound: UnitInRound -> unitInRound.mon.onTurnStart() }
```

Comment:
is this used?

Reply:
Thinking about it now `onTurnStart()` may not make sense